src/app/profile/overview/connected-accounts/connected-accounts.component.html

```diff
@@ -41,10 +41,20 @@ <h2 class="card-pf-title">
                         <span>
```

Comment:
add class `margin-right-15` here to add a nice space between this text and the button group.